README.md

```diff
@@ -1,66 +1,17 @@
-# OCI Artifacts
+# OCI Artifacts (Experimental) with OCI Artifact Reference Support
```

Comment:
the proposed artifact manifest should live in a sub-directory and have it's own readme I think, vs change the purpose of the artifacts repository into a spec repo for the phases of new artifact types the artifact providers are asking for..